.github/workflows/ci.yml

```diff
@@ -7,14 +7,17 @@ on:
       - master
     tags:
       - v*
+  schedule:
+    - cron: '0 0 * * 2'
 
 jobs:
   test:
-    runs-on: ubuntu-latest
     strategy:
       matrix:
+        os: [ubuntu-latest, macos-latest,windows-latest]
```

Comment:
missing space after `macos-latest`

Reply:
Modified this to resemble `go: [ '1.13', '1.17', '>=1.18' ]` which was reviewed and checked in.